src/plugins/intel_gpu/src/graph/impls/ocl/convolution.cpp

```diff
@@ -166,6 +166,30 @@ struct convolution_impl : typed_primitive_impl_ocl<convolution> {
             conv_params.quantization = kernel_selector::QuantizationType::NONE;
         }
 
+        // Swap XY axes
+        if (conv_params.inputs[0].Batch().v == 1 && primitive->deformable_mode == false
+            && conv_params.inputs[0].X().v == 1 && conv_params.inputs[0].Y().v > 1
+            && conv_params.outputs[0].X().v == 1 && conv_params.outputs[0].Y().v > 1
+            && conv_params.weights.X().v == 1 && conv_params.weights.Y().v > 1
+            && conv_params.inputs[0].X().pad.Total() == 0
+            && conv_params.inputs[0].Dimentions() == 4
+            && conv_params.groups != conv_params.inputs[0].Feature().v  // except depth-wise conv
+            && static_cast<int>(conv_params.inputs[0].GetDType()) > 3) { // not support binary/i8/u8 dtype
+            conv_params.inputs[0].SwapXY();
+            conv_params.outputs[0].SwapXY();
+            conv_params.weights.SwapXY();
+            for (auto& desc : conv_params.fused_ops) {
+                desc.output_tensor.SwapXY();
+                for (size_t i = 0; i < desc.tensors.size(); i++) {
+                    desc.tensors[i].SwapXY();
```

Comment:
isn't it necessary to confirm that pad for fused_ops is also zero?

Reply:
Added